roles/openshift_provisioning_aws_facts/README.md

```diff
@@ -0,0 +1,43 @@
+OpenShift Provisioning AWS Facts
+=========
+
+Provides common facts and defaults for other AWS provisioning roles.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+| Name                                   | Default value                                       |                     |
+|----------------------------------------|-----------------------------------------------------|---------------------|
+| openshift_cluster_id                   | default                                             | Cluster ID (combination of openshift_cluster_id and openshift_env_id are expected to be unique for all OpenShift clusters) |
```

Comment:
I don't think this prevents that approach, but we still need a way to do unique naming by default.